cmake/dependencies.cmake

```diff
@@ -30,49 +30,55 @@ set(CFLAGS ${DEPENDENCIES_CFLAGS})
 set(CPPFLAGS ${DEPENDENCIES_CPPFLAGS})
 set(LDFLAGS ${DEPENDENCIES_LDFLAGS})
 
+set(USE_SYSTEM_MKSQUASHFS OFF CACHE BOOL "Use system mksquashfs instead of downloading and building our own")
+
+if(NOT USE_SYSTEM_MKSQUASHFS)
```

Comment:
I'd like to see a warning that the system `mksquashfs` might not work as intended. Only the most recent version *might* work.

Reply:
Added a warning.